OpenBGPD.conf

```diff
@@ -26,17 +26,36 @@ group "blackholes" {
   multihop 63
 
   neighbor $projectton {
-    remote-as 65618
+    remote-as 55016
     descr "Project TON"
   }
 }
 
 ## rules section
 deny quick to group blackholes
-match from group blackholes set {
-  nexthop blackhole
-  }
-allow quick from group blackholes
+match from group blackholes set { nexthop blackhole }
+deny from group blackholes
+
+# 65535:666 = Blackhole
+# This will allow any prefixes with the BLACKHOLE community set:
+allow from group blackholes community BLACKHOLE
+
+# 65332:888 = BOGONs
+# This will allow any prefixes that are BOGONs:
+allow from group blackholes community 65332:888
+
+# 65535:3166 = Country-coded prefixes.
+# You can use the ISO-3166 codes to selectively block prefixes from certain
+# countries.  These prefixes will have two communities; [65535:3166] AND
+# [3166:xxxx] where xxxx is the ISO-3166 country code.  See:
+#   https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
+#
+# Example to block Canadian prefixes:
+#   allow from group blackholes community 3166:124
+#
+# Deny 65535:3166 or you will block EVERYTHING:
+deny from group blackholes community 65535:3166
+
```

Comment:
Does this mean we want to deny traffic from Canada?
Does this mean we want to prevent IPs from Canada being added to the blackholes community?